pkg/util/fsnotification/fsnotification.go

```diff
@@ -0,0 +1,52 @@
+package fsnotification
```

Comment:
Unless we have somebody say otherwise, the documentation for a package
doesn't have to be in a `doc.go` file. Specially for a single file package,
it's enough to put the documentation comment right above the package
statement.
FWIW, Go and godoc doesn't treat files called "doc.go" in any special way.

Rodolfo Carvalho

OpenShift Developer Experience

On Thu, Mar 31, 2016 at 9:53 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/util/fsnotification/fsnotification.go
> <https://github.com/openshift/origin/pull/8268#discussion_r58116983>:
>
> > @@ -0,0 +1,52 @@
> > +package fsnotification
>
> will add doc.go. open to suggestions on the name, but calling it fsnotify
> seemed worse, i think fsnotification was one of clayton's suggestions.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8268/files/c0485dc95526dca3af789f1485642e5324936094#r58116983>
>